addons/default/visiosoft/base-theme/resources/css/gallery.css

```diff
@@ -0,0 +1,66 @@
+.product-hb {
+    font-family: -apple-system,
```

Comment:
Do you need these fonts? are we available?

Reply:
these codes written by @YosefGul 

but these are not needed @metehancelik

Reply:
We already had those blocks in a different file which is in advs-module and I do not remember the name of it now. I just moved them in order to be more systematical.

Reply:
@muammertop @fatihalp I checked the 'font' lines, they are not needed. Will delete and commit again.